core/src/test/java/org/elasticsearch/ExceptionSerializationTests.java

```diff
@@ -32,14 +31,26 @@
 import org.elasticsearch.cluster.block.ClusterBlockException;
 import org.elasticsearch.cluster.metadata.SnapshotId;
 import org.elasticsearch.cluster.node.DiscoveryNode;
-import org.elasticsearch.cluster.routing.*;
+import org.elasticsearch.cluster.routing.IllegalShardRoutingStateException;
```

Comment:
It's due to different IDE settings. Some people have their IDEs set to auto-collapse imports when there are more than a few from the same package, others do not. If you'd like, I can update the pull request to not have this noise?